Pipfile

```diff
@@ -28,6 +28,9 @@ app-common-python = "~=0.1.6"
 prometheus-client = "~=0.11.0"
 prometheus-flask-exporter = "~=0.18.2"
 pyyaml = "*"
+watchtower = ">=1.0.0"
+boto3 = ">=1.16.13"
+botocore = ">=1.19.13"
```

Comment:
@saltgen, I would like to understand - is it necessary to add `botocore` package in Pipfile?

Reply:
Any specific reason for it?

Reply:
Found it, module imported here - https://github.com/RedHatInsights/ros-backend/pull/163/files#diff-15966ba8ae1625348a0b922e1f3cac787080802135b9fba3d0234359381aef46R9